i18n/en.json

```diff
@@ -6136,12 +6136,16 @@
     "translation": "An error occurred while saving the file to {{.Filename}}"
   },
   {
+    "id": "utils.config.supported_available_locales.app_error",
+    "translation": "Unable to load mattermost configuration file:  AvailableLocales must include DefaultClientLocale. Set AvailableLocales to empty string as default value."
+  },
+  {
     "id": "utils.config.supported_client_locale.app_error",
-    "translation": "Unable to load mattermost configuration file:  DefaultClientLocale must be one of the supported locales"
+    "translation": "Unable to load mattermost configuration file:  DefaultClientLocale must be one of the supported locales. Set DefaultClientLocale to en as default value."
   },
   {
     "id": "utils.config.supported_server_locale.app_error",
-    "translation": "Unable to load mattermost configuration file:  DefaultServerLocale must be one of the supported locales"
+    "translation": "Unable to load mattermost configuration file:  DefaultServerLocale must be one of the supported locales. Set DefaultServerLocale to en as default value."
```

Comment:
Same thing here where `Set DefaultServerLocale to en as default value` should be `Adding DefaultServerLocale to AvailableLocales`